src/Algebra/Consequences/Base.agda

```diff
@@ -13,14 +13,24 @@ module Algebra.Consequences.Base
 open import Algebra.Core
 open import Algebra.Definitions
 open import Data.Sum.Base
-open import Relation.Binary.Core
+open import Relation.Binary.Consequences
+  using (mono₂⇒monoˡ; mono₂⇒monoʳ)
+open import Relation.Binary.Core using (Rel)
 open import Relation.Binary.Definitions using (Reflexive)
 
-module _ {ℓ} {_•_ : Op₂ A} (_≈_ : Rel A ℓ) where
+module _ {ℓ} {_∙_ : Op₂ A} (_≈_ : Rel A ℓ) where
 
-  sel⇒idem : Selective _≈_ _•_ → Idempotent _≈_ _•_
+  sel⇒idem : Selective _≈_ _∙_ → Idempotent _≈_ _∙_
   sel⇒idem sel x = reduce (sel x x)
 
+  module Congruence (cong : Congruent₂ _≈_ _∙_) (refl : Reflexive _≈_) where
```

Comment:
Can we de-nest this module for readability and move it above the proofs about selectivity? We could also probably make `{ℓ} {_∙_ : Op₂ A}` variables? I assume you've checked that it requires that `_≈_` be explicit as well?

Reply:
Re: `variable`s... see #2592 ;-) which this PR could be marked as blocking on...
Re: should `_≈_` be implicit or explicit? Good question... (see also monotonicty/congruence lemmas above ;-))